python/ray/rllib/agents/ppo/appo_policy.py

```diff
@@ -219,17 +257,19 @@ def make_time_major(*args, **kw):
             dist_class=Categorical if is_multidiscrete else policy.dist_class,
             valid_mask=make_time_major(mask, drop_last=True),
             vf_loss_coeff=policy.config["vf_loss_coeff"],
-            entropy_coeff=policy.entropy_coeff,
+            entropy_coeff=policy.config["entropy_coeff"],
```

Comment:
are you sure this is right? isn't policy.entropy_coeff a tensor?

Reply:
that's odd, how did format.sh do this?

Reply:
ahhh this was from an old commit! Yes, i changed it to policy.config["entropy_coeff"], since APPO does not work with the entropy coeff schedule.